database/sql/src/main/scala/cromwell/database/slick/tables/WorkflowMetadataSummaryEntryComponent.scala

```diff
@@ -49,6 +49,13 @@ trait WorkflowMetadataSummaryEntryComponent {
       if workflowMetadataSummaryEntry.workflowExecutionUuid === workflowExecutionUuid
     } yield workflowMetadataSummaryEntry)
 
+  val workflowMetadataSummaryEntryExistsForWorkflowExecutionUuid = Compiled(
+    (workflowExecutionUuid: Rep[String]) => (for {
+      summaryEntry <- workflowMetadataSummaryEntries
+      if summaryEntry.workflowExecutionUuid === workflowExecutionUuid
+    } yield summaryEntry).exists
```

Comment:
Why is `.exists` returning a `Rep[String]`?

Reply:
It's not. Why do you think `.exists` is returning a `Rep[String]`?